examples/nodejs_wav/Readme.md

```diff
@@ -0,0 +1,44 @@
+# NodeJS voice recognition example using Mozilla DeepSpeech
+
+Download the pre-trained model (1.8GB):
+
+```
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.4.1/deepspeech-0.4.1-models.tar.gz
+tar xvfz deepspeech-0.4.1-models.tar.gz
+```
+
+Edit references to models path if necessary:
+
+```
+let modelPath = './models/output_graph.pbmm';
+let alphabetPath = './models/alphabet.txt';
+let lmPath = './models/lm.binary';
+let triePath = './models/trie';
+```
+
+Install Sox (for .wav file loading):
+
+```
+brew install sox
+```
+
+Install NPM dependencies:
+
+```
+npm install
+```
+
+Run:
+
+```
+node index.js
+```
+
+Result should be something like:
+
+```
+2019-02-22 13:13:44.271363: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
+audio length 2.0241875
+result: more lurtee speech
```

Comment:
Using one of the samples or the `audio-0.4.1.tar.gz` has the benefit of showing a better result, `more lurtee speech` kinda looks like the engine is working badly, which is not something we want people to think of as a first experience.

People already have to download the model, it'd be really nicer to refer to the `audio-0.4.1.tar.gz` here.

Reply:
Yeah that is better. I updated the Readme to reflect that and I added an optional argument back in so it's a bit more like how the original client.js was intended to work.